mlir/lib/Dialect/Rock/Transforms/BlockwiseGemmToThreadwise.cpp

```diff
@@ -15,38 +15,38 @@
 // limitations under the License.
 // ============================================================
 //
-// This pass converts miopen.blockwise_* ops to miopen.threadwise_*
+// This pass converts rock.blockwise_* ops to rock.threadwise_*
 // and lowers other higher-level ops like transform and fill in preparation for
 // the threadwise lowering
 //
 //===-----------------------------------------------------===//
 #include "PassDetail.h"
 
 #include "mlir/Dialect/Arithmetic/IR/Arithmetic.h"
-#include "mlir/Dialect/MIOpen/MIOpen.h"
-#include "mlir/Dialect/MIOpen/Passes.h"
-#include "mlir/Dialect/MIOpen/TransformMapBuilder.h"
-#include "mlir/Dialect/MIOpen/utility/builderUtils.h"
-#include "mlir/Dialect/MIOpen/utility/transformMapUtils.h"
+#include "mlir/Dialect/Rock/IR/Rock.h"
+#include "mlir/Dialect/Rock/Passes.h"
+#include "mlir/Dialect/Rock/IR/TransformMapBuilder.h"
+#include "mlir/Dialect/Rock/utility/builderUtils.h"
+#include "mlir/Dialect/Rock/utility/transformMapUtils.h"
 
 #include "mlir/Dialect/Affine/IR/AffineOps.h"
-#include "mlir/Dialect/MIOpen/XdlopsCodeSelection.h"
+#include "mlir/Dialect/Rock/IR/XdlopsCodeSelection.h"
 #include "mlir/Dialect/MemRef/IR/MemRef.h"
 #include "mlir/Transforms/DialectConversion.h"
 
 #include "llvm/ADT/SmallVector.h"
 #include "llvm/Support/Debug.h"
 
-#define DEBUG_TYPE "miopen-blockwise-to-threadwise"
+#define DEBUG_TYPE "rock-blockwise-to-threadwise"
```

Comment:
Can we go ahead and make this `rock-blockwise-gemm-to-threadwise` to match the pass name?

Reply:
Might pass on these since I need to wrap this up to unblock everyone